infrastructure/README.md

```diff
@@ -49,6 +49,26 @@ To connect from SDK or REST we need to know the network address and authorizatio
     cat $ALGORAND_DATA/algod.net
     cat $ALGORAND_DATA/algod.token
 
+## Using Sandbox 2.0
+
+Algorand Sandbox is a fast way to create and configure an Algorand development environment with [Algod](https://github.com/algorand/go-algorand) and [Indexer](https://github.com/algorand/indexer). To quickly bring up a private network with algorand sandbox and use it within algob, following jobs are provided (in `/infrastructure`) :-
+*NOTE:* Please make sure to have [Docker Compose](https://docs.docker.com/compose/install/) installed (with non root privilages) before running sandbox.
+1. `sandbox-up` - Clones the sandbox [git repo](https://github.com/algorand/sandbox.git) in `~/sandbox` and setups up the network (this might take a while).
```

Comment:
```suggestion
1. `sandbox-up` - Clones the sandbox [git repo](https://github.com/algorand/sandbox.git) in `~/sandbox` and setups the network (this might take a while).
```

Reply:
**setups and starts the network.

Reply:
ah, yes.